continuation-local-storage/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for continuation-local-storage v3.2.0
+// Project: https://github.com/othiym23/node-continuation-local-storage
+// Definitions by: Jang-Ho Hwang <https://github.com/rath>, Kei Son <https://github.com/heycalmdown>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare module ContinuationLocalStorage {
```

Comment:
it leads to add too many export keyword. but if the style is preferred I will revision it that way.